docs/getting-started/prerequisites.md

```diff
@@ -78,6 +78,7 @@ Our supported systems include:
 - **Ubuntu** (14.04, 16.04, 17.10, 18.04, 18.10)
 - **Debian** 8 (jessie) or later
 - Recent versions of **macOS** with [homebrew] (experimental)
+- **Fuchsia OS** 
```

Comment:
This documentation refers to the host system to develop / test ClusterFuzz on. The Fuchsia support that was added relatively recently is for fuzzing bots and it's not a native Fuchsia support, it uses QEMU on Linux VMs.

Reply:
Thanks for making it clear!